examples/java/spring-boot/src/test/java/uk/gov/api/springboot/config/CorrelationIdFilterTest.java

```diff
@@ -0,0 +1,83 @@
+package uk.gov.api.springboot.config;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.mockito.Mockito.*;
+
+import java.io.IOException;
+import java.util.UUID;
+import javax.servlet.FilterChain;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+class CorrelationIdFilterTest {
+
+  @Mock private HttpServletRequest request;
+  @Mock private HttpServletResponse response;
+  @Mock private FilterChain filterChain;
+
+  private final CorrelationIdFilter filter = new CorrelationIdFilter();
+
+  @Test
+  void delegatesToFilterChain() throws ServletException, IOException {
+    filter.doFilterInternal(request, response, filterChain);
+
+    verify(filterChain).doFilter(request, response);
+  }
+
+  @Test
+  void validCorrelationIdIsAddedToResponse() throws ServletException, IOException {
```

Comment:
At this point, we shouldn't be calling it `valid`, because there's no validation being performed